data/json/items/book/fabrication.json

```
@@ -328,7 +328,7 @@
     "skill": "fabrication",
     "required_level": 5,
     "max_level": 10,
-    "proficiencies": [ { "proficiency": "prof_bladesmith", "time_factor": 0.65, "fail_factor": 0.6, "include_prereqs": false } ],
+    "proficiencies": [ { "proficiency": "prof_bladesmith", "time_factor": 0.2, "fail_factor": 0.3, "include_prereqs": false } ],
```

```suggestion
    "proficiencies": [ { "proficiency": "prof_bladesmith", "time_factor": 0.25, "fail_factor": 0.3, "include_prereqs": false } ],
```
"Step-by-step photo instructions", plus consideration of other stats.